scan/spec/test_command_generator_spec.rb

```diff
@@ -1,5 +1,8 @@
 describe Scan do
+  require "rspec/mocks/standalone"
```

Comment:
Without this line I was getting these errors:

```
  1) Scan Scan::TestCommandGenerator raises an exception when project path wasn't found
     Failure/Error: allow(FastlaneCore::FastlaneFolder).to receive(:path).and_return(".")
       The use of doubles or partial doubles from rspec-mocks outside of the per-test lifecycle is not supported.
     # ./scan/spec/test_command_generator_spec.rb:4:in `block (2 levels) in <top (required)>'
```

After googling around I've found 3 possible solutions. 2 of those would need a `require` line and the third one was involving use of `RSpec::Mocks.with_temporary_scope do/end` wrapper, but it didn't work for me.

http://stackoverflow.com/questions/21235269/method-stubbing-on-beforeall